### DISCUSSION TITLE
无法连接aws.ec2在使用证书的情况下.

### DISCUSSION THREAD

**Discussion by AiMonRen** [electerm/electerm#3061] [Q&A]
ec2-xx-xxx-xxx-xxx.xx-xxxxxxxxx-1.compute.amazonaws.com 这种登陆ip  用户名 ec2-user 
使用私钥连接.确定aws默认是没有私钥密码.在这种情况下点解测试连接 报错connection fails: test failed
如果直接保存连接..他会向我 索要 私钥密码.点击忽略.-> 无效 最后连接失败
在electerm的个人终端界面..输入如下命令
ssh -i  ./xxxxxx.pem  ec2-xx-xxx-xxx-xxx.xx-xxxxxxxxx-1.compute.amazonaws.com
可以正常登录服务器.确定不是,服务器的问题以及证书或者ip的问题

**Comment by zxdong262** [comment]
请问是怎么生成的私钥，确实某些格式的可能不支持

**Reply by AiMonRen** [reply]
私钥的话.是在购买aws ec2时,云厂商自动生成并且提供的

**Comment by zxdong262** [comment]
可以试试最新版吗 https://github.com/electerm/electerm/releases/tag/v1.37.80

**Comment by ParseC-ctrl** [comment]
最新版已修复
